docs/words.tex

```diff
@@ -380,6 +399,8 @@ \section{Disk I/O}
 \item[saveb ( start end filenameptr filenamelength -- )] Save binary block.
 \item[device ( device\# -- )] Switches the active device.
 \item[save-forth filename ( -- )] Saves the forth to the given filename.
+\item[save-pack filename ( -- )] Saves a compact version of forth to the given filename.

```

Comment:
Could add a reference to "turn-key operation" section which explains in further detail what actually happens. Same for save-prg.